src/main/java/seedu/address/logic/parser/RecipeBookParserUtil.java

```diff
@@ -0,0 +1,170 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.StringUtil;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.recipe.CompletionTime;
+import seedu.address.model.recipe.Ingredient;
+import seedu.address.model.recipe.Name;
+import seedu.address.model.recipe.ServingSize;
+import seedu.address.model.recipe.Step;
+
+/**
+ * Contains utility methods used for parsing strings in the various *Parser classes.
+ */
+public class RecipeBookParserUtil {
+
+    public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero positive integer.";
+    public static final String MESSAGE_MISSING_INGREDIENT_FIELDS =
+            "Ingredient is not in the <name> <quantity> <quantifier> format.";
+    public static final String MESSAGE_INVALID_PORTION = "ServingSize is not a non-zero positive number.";
```

Comment:
Perhaps we can update this to use the Serving size name instead?

```suggestion
    public static final String MESSAGE_INVALID_SERVING_SIZE = "ServingSize is not a non-zero positive number.";
```

Reply:
Yeah I agree with Aidan since we've refactored `Portion` to `ServingSize`